README.md

```diff
@@ -1,15 +1,418 @@
+    {<id1>: {"mappings" [{"ns": <namespace1_1>, "id": <id1_1>},
+                          ...
+                         {"ns": <namespace1_N>, "id": <id1_N>}
+                         ]
+             },
+     ...
+     <idN>: {"mappings" [{"ns": <namespaceN_1>, "id": <idN_1>},
+                          ...
+                         {"ns": <namespaceN_N>, "id": <idN_N>}
+                         ]
+             }
+     }
+else:
+    {<id1>: {"admin": [{"ns": <namespace1_1>, "id": <id1_1>},
+                        ...
+                       {"ns": <namespace1_N>, "id": <id1_N>}
+                       ],
+             "other": [{"ns": <namespace1_N+1>, "id": <id1_N+1>},
+                        ...
+                       {"ns": <namespace1_N+M>, "id": <id1_N+M>}
+                      ]
+             },
+      ...
+     <idN>: {"admin": [{"ns": <namespaceN_1>, "id": <idN_1>},
+                        ...
+                       {"ns": <namespaceN_N>, "id": <idN_N>}
+                       ],
+             "other": [{"ns": <namespaceN_N+1>, "id": <idN_N+1>},
+                        ...
+                       {"ns": <namespaceN_N+M>, "id": <idN_N+M>}
+                      ]
+             },
+     }
+```
+
+A maximum of 1000 ids may be supplied.
+
+The mappings in the `admin` key are mappings where the provided half of the mapping
+is the administrative half - e.g. the namespace in the url is the administrative namespace
+in the mapping. Mappings in the `other` key denote mappings where the provided half of the
+mapping is not the administrative half.
+
+### Delete mappings
+
+```
+HEADERS:
+Authorization: [Auth source] <token>
+
+DELETE /api/v1/mapping/<administrative namespace>/<namespace>/
+{<administrative id1>: <id1>,
+ ...
+ <administrative idN>: <idN>
+ }
+```
+
+A maximum of 10000 ids may be supplied.
+
+## Developer notes
+
+### Adding and releasing code
+
+* Adding code
+  * All code additions and updates must be made as pull requests directed at the develop branch.
+    * All tests must pass and all new code must be covered by tests.
+    * All new code must be documented appropriately
+      * Sphinx code documentation
+      * General documentation if appropriate
+      * Release notes
+* Releases
+  * The master branch is the stable branch. Releases are made from the develop branch to the master
+    branch.
+  * Update the version as per the semantic version rules in `src/jgikbase/idmapping/service/mapper_service.py`.
+  * Tag the version in git and github.
+
+### Running tests
+
+* Copy `test.cfg.example` to `test.cfg` and fill in the values appropriately.
+  * If it works as is start buying lottery tickets immediately.
+* `make test`
+
+### UI
+
+Most text fields are arbitrary text entered by a data uploader. These fields should be
+HTML-escaped prior to display.
+  
+Use common sense when displaying a field from the server regarding whether the field should be
+html escaped or not.
+  
+### Exception mapping
```

Comment:
Content here looks good to me, I would just rorder things. As a client to this service I would expect to see:
* Overview -- what it does
* What to pass for auth (brief info about auth)
* The API
* Sysadmin commands
* How to build and run
* Information about how it was implemented and how it can be expanded